static/app/views/dashboards/manage/dashboardTable.tsx

```diff
@@ -50,6 +53,55 @@ enum ResponseKeys {
   OWNER = 'createdBy',
   ACCESS = 'permissions',
   CREATED = 'dateCreated',
+  FAVORITE = 'isFavorited',
+}
+
+type FavoriteButtonProps = {
+  api: Client;
+  dashboardId: string;
+  isFavorited: boolean;
+  organization: Organization;
+};
+
+function FavoriteButton({
+  isFavorited,
+  api,
+  organization,
+  dashboardId,
+}: FavoriteButtonProps) {
+  const [favorited, setFavorited] = useState(isFavorited);
+  return (
+    <Feature features="dashboards-favourite">
+      <StyledFavoriteButton
+        aria-label={t('Favorite Button')}
+        data-test-id="xx"
```

Comment:
if this isn't being used can we take this out? or make it more comprehensive?

Reply:
oh oops I was using it for testing something